WiserTaskScheduler/AutoUpdater/Services/UpdateService.cs

```diff
@@ -353,12 +369,34 @@ private void RevertUpdate(WtsModel wts, ServiceController serviceController, Ver
             // Try to start the previous installed version again.
             serviceController.Start();
             serviceController.WaitForStatus(ServiceControllerStatus.Running);
-            EmailAdministrator(wts.ContactEmail, "WTS Auto Updater - Updating failed!", $"Failed to update WTS '{wts.ServiceName}' to version {versionToUpdateTo}, successfully restored to version {currentVersion}.<br/><br/>Error when updating:<br/>{updateException.ToString().ReplaceLineEndings("<br/>")}", wts.ServiceName);
+
+            var subject = "WTS Auto Updater - Updating failed!";
+            var message= $"Failed to update WTS '{wts.ServiceName}' to version {versionToUpdateTo}, successfully restored to version {currentVersion}.<br/><br/>Error when updating:<br/>{updateException}";
+            
+            logger.LogError(message);
+            InformPeople(wts,subject,message);
         }
         catch (InvalidOperationException revertException)
         {
-            logger.LogError($"Failed to update WTS '{wts.ServiceName}' to version {versionToUpdateTo}, failed to restore version {currentVersion}.{Environment.NewLine}{Environment.NewLine}Error when reverting:{Environment.NewLine}{revertException}{Environment.NewLine}{Environment.NewLine}Error when updating:<br/>{updateException}");
-            EmailAdministrator(wts.ContactEmail, "WTS Auto Updater - Updating and reverting failed!", $"Failed to update WTS '{wts.ServiceName}' to version {versionToUpdateTo}, failed to restore version {currentVersion}.<br/><br/>Error when reverting:<br/>{revertException.ToString().ReplaceLineEndings("<br/>")}<br/><br/>Error when updating:<br/>{updateException.ToString().ReplaceLineEndings("<br/>")}", wts.ServiceName);
+            var subject = "WTS Auto Updater - Updating and reverting failed!";
+            var message= $"Failed to update WTS '{wts.ServiceName}' to version {versionToUpdateTo}, failed to restore version {currentVersion}.{Environment.NewLine}{Environment.NewLine}Error when reverting:{Environment.NewLine}{revertException}{Environment.NewLine}{Environment.NewLine}Error when updating:{Environment.NewLine}{updateException}";
+            
+            logger.LogError(message);
+            InformPeople(wts,subject,message);
+        }
+    }
+
+    private void InformPeople(WtsModel wts, string subject, string message, bool sendEmail = true, bool sendSlack = true)
+    {
+        if (sendEmail)
+        {
+            var emailMessage = message.Replace(Environment.NewLine, "<br/>");
+            EmailAdministrator(wts.ContactEmail,subject,emailMessage,wts.ServiceName);
```

Comment:
I'm not going to reject the PR for this, but please separate parameters with spaces.